# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vim/desc.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Yet again an update vim .desc rule patch. This options exist a bit longer
though, but I forgot to include them in the patch I submitted upstream for
the 7.0 version ... (there are still some missing, but they are mostly
historical and might be dropped sooner than later).

  - Rene Rebe <rene@exactcode.de>

--- vim70/runtime/syntax/desc.vim.orig	2006-07-01 17:23:24.000000000 +0000
+++ vim70/runtime/syntax/desc.vim	2006-07-01 17:27:39.000000000 +0000
@@ -14,7 +14,7 @@
 " syntax definitions
 
 setl iskeyword+=-
-syn keyword descFlag DIETLIBC KAFFE JAIL FPIC-QUIRK NO-LIBTOOL-QUIRK AUTOMAKE-QUIRK NO-AS-NEEDED NO-SSP contained
+syn keyword descFlag DIETLIBC KAFFE JAIL NOPARALLEL FPIC-QUIRK LIBTOOL-WUIRK NO-LIBTOOL-FIX AUTOMAKE-QUIRK NO-AS-NEEDED NO-SSP KERNEL INIT LIBC CC CXX F77 KCC contained
 syn keyword descLicense Unknown GPL LGPL FDL MIT BSD OpenSource Free-to-use Commercial contained
 
 " tags
